addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/handler/HomematicThingHandler.java

```diff
@@ -111,6 +100,120 @@ public void run() {
         });
     }
 
+    private void doInitializeInBackground()
```

Comment:
Small semantic comment this method is not the reason it's running in the background, but it's wrapped a few lines up that is why its running in the background.

Reply:
That's right. With the method naming I intended to make clear in what scenario the method is supposed to be run in.

Reply:
(Having said that, I'm fine with removing the InBackground suffix if you prefer)